src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -284,7 +281,7 @@ public class LDAPSecurityRealm extends AbstractPasswordBasedSecurityRealm {
 
     /**
      * Query to locate an entry that identifies the user, given the user name string.
-     *
+     * <p>
```

Comment:
this change and many like it do not seem correct to me - `Normally "uid={0}"` is not a paragraph.  It is strongly related to the previous sentence and to me does not seem to make grammatical sense to me.
I have not requested changes or approved as I was ambivalent on this, and as such decided to let the reset of the maintainers make a decision.

There is no generally need AFAICT to separate the introductory sentence from any following ones.  they often can and do make sense together.

Reply:
A p-element for separating examples or snippets is not uncommon, e.g. in Javadoc of Java classes it is occasionally seen.